src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter_nodelts.py

```diff
@@ -96,7 +86,7 @@ def test_containerapp_session_code_interpreter_nodelts_e2e(self, resource_group)
             identifier_name,
             "cert.txt"),
             checks=[
-            JMESPathCheck('properties.filename', 'cert.txt'),
+            JMESPathCheck('name', 'cert.txt'),
```

Comment:
such change is breaking for cli users. Can  we make sure there is no change for cli command output?

Reply:
Even there is breaking change in swagger, we should still avoid breaking change in cli commands. Discussed with hong, given there is no cli usage, we could allow cli breaking change this time. But in the future, with increasing cli users, such breaking change is not allowed.

Reply:
Thank you, understood, will keep in mind for this in the future.